bgpd/bgp_route.c

```diff
@@ -4454,19 +4455,23 @@ void bgp_announce_route_all(struct peer *peer)
 		bgp_announce_route(peer, afi, safi);
 }
 
-static void bgp_soft_reconfig_table(struct peer *peer, afi_t afi, safi_t safi,
-				    struct bgp_table *table,
-				    struct prefix_rd *prd)
+static void bgp_soft_reconfig_table_index(struct peer *peer, afi_t afi,
+					  safi_t safi, struct bgp_table *table,
+					  struct prefix_rd *prd,
+					  uint32_t min_idx, uint32_t max_idx)
 {
+	uint32_t idx;
 	int ret;
 	struct bgp_dest *dest;
 	struct bgp_adj_in *ain;
 
 	if (!table)
 		table = peer->bgp->rib[afi][safi];
 
-	for (dest = bgp_table_top(table); dest; dest = bgp_route_next(dest))
-		for (ain = dest->adj_in; ain; ain = ain->next) {
+	for (idx = min_idx, dest = bgp_table_top(table);
```

Comment:
What's the purpose of min_idx?  Why do we need it?

Reply:
was a mistake